src/combinatorics/simple_graph/basic.lean

```diff
@@ -82,6 +101,14 @@ The edges of G consist of the unordered pairs of vertices related by
 -/
 def edge_set : set (sym2 V) := sym2.from_rel G.sym
 
+/--
+The `incidence_set` is the set of edges incident to a given vertex.
+-/
+def incidence_set (v : V) : set (sym2 V) := {e ∈ G.edge_set | v ∈ e}
+
+lemma incidence_set_subset (v : V) : G.incidence_set v ⊆ G.edge_set :=
+by tidy
```

Comment:
```lean
λ _ h, h.1
```

Reply:
For some reason this doesn't work on my end, Lean doesn't understand what type the underscore is supposed to be

Reply:
Ok, then ignore that.  It's probably too brittle of a term-mode proof anyway since it depends on the implementation of `{e ∈ G.edge_set | v ∈ e}`.

Reply:
This one is less sketchy than the term mode proof I gave before:
```lean
lemma incidence_set_subset (v : V) : G.incidence_set v ⊆ G.edge_set :=
λ _ h, (set.mem_sep_iff.mpr h).1
```

Reply:
I'm a fool, I forgot to delete the `by`. The first proof works